src/controllers/healthRecord.controller.js

```diff
@@ -0,0 +1,108 @@
+const { isAdmin } = require('../middlewares/authJwt');
+const healthRecord = require('../models/healthRecord.model');
+
+exports.createHealthrecord = async (req, res) => {
+
+    try {
+
+        const orderObj = {
+            weight : req.body.weight,
+            height : req.body.height,
+            bloodPressure : req.body.bloodPressure,
+            sugerLevel : req.body.sugerLevel,
+            tempearture : req.body.tempearture,
+            symptoms : req.body.symptoms
+        }
+
+        const healthRecordCreated = await Order.create(orderObj);
+
+        const user = req.user
+        user.healthRecords.push(healthRecord._id);
+        await user.save();
+
+        res.status(201).send(healthRecordCreated);
+    } catch (err) {
+        console.log("Error while doing the DBb operations", err.message);
+        res.status(500).send({
+            message : "Internal server error"
+        })
+    }
+}
+
+exports.getAllhealthRecords = async (req, res) => {
+
+    try {
+
+        if(isAdmin){
```

Comment:
reverse has been done. If isAdmin is true, we should return all the healthRecords